test/unit/node_test.rb

```diff
@@ -58,6 +58,16 @@ class NodeTest < ActiveSupport::TestCase
       assert_equal nodes.length, Tag.find_nodes_by_type('awesome', ['note', 'page']).length
     end
   end
+  
+  test 'node mysql native fulltext search returning two tag-based matches' do
+    assert Node.count > 0
+    if ActiveRecord::Base.connection.adapter_name == 'Mysql2'
+      nodes = Node.search(query: 'awesome cool' limit: 1000)
+      assert_not_nil nodes
+      assert_equal 1, nodes.length
+      assert_equal nodes(:awesome_cool), nodes.first
```

Comment:
OK almost there! So now, you have to create an entry in the `fixtures/node_tags.yml` file to link your new `cool` tag to just one node that already has the `awesome` tag linked. Then you'll have just a single node with both tags linked. Make sense? Great progress here!

Reply:
@jywarren not sure I understand. Do you mean like:
```
awesome_cool:
  tid:22
  name: awesome
  name: cool
```
or
```
 awesome_cool:
  tid: 22
  name: awesome, cool
```
or do I add parent: awesome_cool to each awesome and cool tag?

Reply:
Well I think theres already an awesome tag so you can just add cool. But in the other file, /test/fixtures/node_tags.yml, each entry links a tag to a node. So you'll need to add an entry there to ensure both distinct tags from this file are linked to a single node. Make sense?

Reply:
@jywarren I think I get it! Check out my recent changes

Reply:
@jywarren I don't know why I'm getting these errors.... :/

Reply:
OK, your tags are great now. You've linked your `cool` tag to node with id=1. So on this line now, you'll want to assert that it's the same as this node: `nodes(:one)` -- that's this one: https://github.com/publiclab/plots2/blob/master/test/fixtures/nodes.yml#L1-L11

Reply:
Are you able to read the errors? If you click the little X next to your commit, you'll be able to read the full output. In this case, it looks like you may be missing a parenthesis?

```
/usr/local/bundle/gems/skylight-core-3.1.2/lib/skylight/core/probes.rb:119:in `require': /app/test/unit/node_test.rb:65: syntax error, unexpected tIDENTIFIER, expecting ')' (SyntaxError)
ch(query: 'awesome cool' limit: 1000)
```

https://travis-ci.org/publiclab/plots2/builds/465228995#L2953-L2954

Reply:
I was missing a comma! That's why it expected the bracket to end after 'awesome cool'

Reply:
Ok, @jywarren I think it's good now :)